insights/combiners/krb5.py

```diff
@@ -0,0 +1,83 @@
+"""
+krb5 configuration
+==================
+The krb5 files are normally available to rules as a list of
+Krb5Configuration objects. This combiner turns those into one
+set of data, and provides a ``get()`` method to get value.
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.krb5 import Krb5Configuration
+
+
+@combiner(requires=[Krb5Configuration])
+class AllKrb5Conf(object):
+    """
+    Combiner for accessing all the krb5 configuration files, the format is dict.
+    There may be multi files for krb5 configuration, and the main config file is
+    krb5.conf. In the situation that same section is both in krb5.conf and other
+    configuration files, section in krb5.conf is the available setting. Data from
+    parser krb5 is list of dict(s), this combiner will parse this list and return
+    a dict which containing all valid data.
+    """
+
+    def __init__(self, local, shared):
+        self.data = {}
+        main_data = {}
+        self.includedir = []
+        self.include = []
+        self.module = []
```

Comment:
Please include docstrings for the attributes `includedir`, `include`, and `module`.